### ISSUE TITLE
JLCPCB CPL Missing data

### ISSUE THREAD

**Issue by statikg11** [jfedor2/flatbox#56]
When uploading cpl file on JLCPCb error message below is generating. any solutions?

![JLCPCB](https://github.com/jfedor2/flatbox/assets/169212163/86137a11-5c79-4c29-bffb-0be3ef1b11e7)

**Comment by jfedor2** [comment]
These are the keyboard and tact switches, they're not assembled by JLCPCB, so just ignore it and continue.

(They shouldn't be in the BOM file in the first place.)

**Comment by statikg11** [comment]
so what files should i drop here
![bom cpl](https://github.com/jfedor2/flatbox/assets/169212163/eb90ad30-4356-4751-a1be-fe81efcdb4c8)

**Comment by jfedor2** [comment]
I just meant I shouldn't have included those entries in that file. Use the same files and just click "continue".

**Comment by statikg11** [comment]
Ok so use the cpl file twice?

**Comment by jfedor2** [comment]
No, use BOM and CPL files as you did and just click "continue".

**Comment by statikg11** [comment]
Thank you. I was able to upload all required files turns out it was an error with the website. Customer support was able to troubleshoot it and resolve.
